.github/workflows/push.yml

```diff
@@ -0,0 +1,43 @@
+on: push
```

Comment:
If you want to drop CircleCI. You can see an example of the results of the tests here https://github.com/hellvinz/influxdb-client-ruby/runs/661529686